test/extended/openstack/cpms.go

```diff
@@ -43,13 +43,9 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] ControlPlane Mach
 		skipUnlessMachineAPIOperator(ctx, dc, clientSet.CoreV1().Namespaces())
 		controlPlaneMachineSet, numCpms, err = getControlPlaneMachineSet(ctx, dc)
 		if err != nil {
-			if numCpms == 0 {
-				// In some cases (e.g. UPI), the control plane machine set is not created.
-				e2eskipper.Skipf("Failed to get control plane machine sets: %v", err)
-			} else {
-				o.Expect(err).NotTo(o.HaveOccurred())
-			}
+			e2eskipper.Skipf("Failed to get control plane machine sets: %v", err)
```

Comment:
If any error happen we would skip the test, is this really desired?

Reply:
yes, we rely on CPMS data, so let's skip the test if we can't get it.

Reply:
disregard my comment. I didn't see your last comment before I commented